### PR TITLE
Add react-date-range v1 support(remove moment)

### PR DESCRIPTION
**Pull request by Curzy**
Remove moment dependency & types by react-date-range v1
---
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/hypeserver/react-date-range/releases/tag/v1.0.0-alpha0>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/react-date-range/index.d.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
-// Type definitions for react-date-range 0.95
+// Type definitions for react-date-range 1.0
 // Project: https://github.com/Adphorus/react-date-range/
 // Definitions by: Junbong Lee <https://github.com/Junbong>
 //                 John Demetriou <https://github.com/DevsAnon>
+//                 Minseok Choi <https://github.com/Curzy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.0
```
Comment (RyanCavanaugh) [inline types/react-date-range/index.d.ts:7]: This file doesn't appear to use any features from 4.0 - this should be the *minimum TS version needed to support this file*, not the version you wrote the file with.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/react-date-range/index.d.ts (hunk 1 above)
Comment (Curzy) [reply types/react-date-range/index.d.ts]: Thank you for review. i returned TypeScript version to 2.8

**Review by DemetriouJohn — APPROVED** [review]

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by andrewbranch** [comment]
@typescript-bot ☝️ stop snoozing on the job

**Comment by Curzy** [comment]
> @Curzy Everything looks good here. Great job! I am ready to merge this PR on your behalf.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! ❤️
> 
> (@Junbong, @DevsAnon: you can do this too.)

Ready to merge

**Comment by Curzy** [comment]
> Ready to merge

**Comment by Curzy** [comment]
Ready to merge

**Pull request merged**